media/js/base/sentry-consent.es6.js

```diff
@@ -59,7 +59,7 @@ const SentryConsent = {
                 dedupeIntegration(),
                 globalHandlersIntegration(),
                 httpContextIntegration(),
-                inboundFiltersIntegration(options)
```

Comment:
this popped up as deprecated in VSCode linting, it seems a simple name swap with no functional changes, but I'm not sure how to test